### PR TITLE
Handle tombstone event

### PR DESCRIPTION
**Pull request by kgalieva**
Fixes 
```
bundler: failed to load command: bin/avro-example (bin/avro-example)
/Users/lourens/.gem/ruby/3.1.2/gems/rdkafka-0.12.0/lib/rdkafka/consumer.rb:548:in `block in each_batch': undefined method `bytesize' for nil:NilClass (NoMethodError)

          bytes += message.payload.bytesize
                                  ^^^^^^^^^
	from /Users/lourens/.gem/ruby/3.1.2/gems/rdkafka-0.12.0/lib/rdkafka/consumer.rb:529:in `loop'
	from /Users/lourens/.gem/ruby/3.1.2/gems/rdkafka-0.12.0/lib/rdkafka/consumer.rb:529:in `each_batch'
```

### CODE REVIEW

**Comment by kgalieva** [comment]
The jruby test failure is unrelated to this change
```
Rdkafka::Config configuration should allow configuring zstd compression
     # Zstd compression not supported on this machine
     Failure/Error: raise ConfigError.new(error_buffer.read_string)
  
     Rdkafka::Config::ConfigError
       Unsupported value "zstd" for configuration property "compression.codec": libzstd not available at build time
```

**Comment by methodmissing** [comment]
@thijsc 🧇  :wave: long time! if and when there's a free moment, mind having a quick :eyes: ? We have a couple of PRs incoming from Shopify

**Review by thijsc — APPROVED** [review]

**Comment by thijsc** [comment]
Long time no see! We've been chasing a segfault that sometimes happens on exit which has been messing with the CI builds, that's been holding up the flow a bit. I hope we can resolve that shortly and then we will get more speed again.

**Comment by methodmissing** [comment]
Happy to allocate some time this evening towards this one and see if we can unblock:

> Long time no see! We've been chasing a segfault that sometimes happens on exit which has been messing with the CI builds, that's been holding up the flow a bit. I hope we can resolve that shortly and then we will get more speed again.

**Comment by thijsc** [comment]
That could be very helpful! The issue is tracked in #216. We just got a reply with some hints from the librdkafka maintainer in https://github.com/edenhill/librdkafka/issues/3981#issuecomment-1265334684

**Comment by thijsc** [comment]
I triggered a rerun. The zstd spec fail is pending, we can ignore that one.

**Comment by thijsc** [comment]
Could you rebase on main? A fix was merged that will hopefully lead to a green build for this.

**Comment by thijsc** [comment]
That's a pending spec actually, it should not fail the build.

**Comment by thijsc** [comment]
Oh we did get a segfault here again `Segmentation fault (core dumped)`. Let me see if I can reproduce it here, been chasing this one for a long time :-).

**Comment by mensfeld** [comment]
There's one more related to some sort of linked list that occurs once in a while.

**Comment by thijsc** [comment]
Of course when I try again it's gone :-). I've retriggered the build, will merge soon.

**Pull request merged**
